thoth/user_api/api_v1.py

```diff
@@ -414,6 +414,26 @@ def list_python_package_indexes():
     return graph.python_package_index_listing()
 
 
+def list_hardware_environments(page: int = 0,):
+    """List hardware environments in the graph database."""
+    graph = GraphDatabase()
+    graph.connect()
+    return {
+        "parameters": {"page": page,},
+        "hardware_environments": graph.get_hardware_environments(is_external, offset=page),
```

Comment:
`is_external` is now not declared. Use `is_external=False`.

Reply:
:100: great catch!